src/test/java/bio/terra/service/search/SearchServiceTest.java

```diff
@@ -133,19 +133,23 @@ public void querySnapshotTest() throws Exception {
         SearchHits mockHits = mock(SearchHits.class);
         SearchHit mockHit = mock(SearchHit.class);
         when(mockHits.iterator()).thenReturn(Arrays.stream(new SearchHit[]{mockHit}).iterator());
-        when(mockHit.getSourceAsMap()).thenReturn(Map.of("testKey", "testValue"));
+        when(mockHit.getSourceAsMap()).thenReturn(Map.of(
+            "uuid", "41b203f2-c7a5-4232-bcb3-f3442ba9baaf",
+            "tim__examplec__identifierp__now", "0"
```

Comment:
In case we change the way encoding works someday, I think it would be better to use the `encode` method here to create the encoded name.